py4DSTEM/io/__init__.py

```diff
@@ -1,4 +1,5 @@
 from . import datastructure
+from .datastructure import DataCube, PointListArray, PointList
```

Comment:
I changed it to:

from .datastructure import *

Is there any risk in importing all of them?

Reply:
I think it is fine

Reply:
Made the change and pushed it